src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/ICustomMarshalerTests.cs

```diff
@@ -0,0 +1,537 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using Xunit;
+
+namespace System.Runtime.InteropServices.Tests
+{
+    public class ICustomMarshalerTests
+    {
+        // To avoid having to create a native test library to reference in tests that
```

Comment:
@yizhang82 Yes, @shrah added it , but the ICustomMarhsaller  was included as a type in CoreFX contract , where it should be a type forwarder to corelib. @shrah fixed it in corefx , but the updated S.R.Interop contract was not yet available in  CoreCLR when i added the test.